crates/runtime/tests/dynamodb/mod.rs

```diff
@@ -52,18 +52,6 @@ async fn dynamodb_federated() -> Result<(), anyhow::Error> {
                 () = cloned_rt.load_components() => {}
             }
 
-            let mut query_result = rt
```

Comment:
Unrelated: this logic was perviously moved into `run_and_snapshot_query` but the code has not been removed.